core/storage/classifier/gae_models.py

```diff
@@ -111,7 +111,7 @@ def get_export_policy(cls) -> Dict[str, base_models.EXPORT_POLICY]:
         })
 
     @classmethod
-    def _generate_id(cls, exp_id: str) -> str:
+    def get_entity_id(cls, exp_id: str) -> str:
```

Comment:
Why is this changed? "generate" seems relevant here since it's not deterministic.